Server/prisma/schema.prisma

```diff
@@ -1,114 +1,98 @@
+// Prisma: Generator for typescript code
 generator client {
   provider = "prisma-client-js"
 }
 
+// Prisma: Connection to the database
 datasource db {
   provider = "postgresql"
   url      = env("DATABASE_URL")
 }
 
 // ============================================================================ //
-// Models
-// Available Models are:
-//      - User
-//      - Tracker
-//      - Log (of Tracker)
-//      - VehicleType
-//      - Vehicle
-//      - Track
-//      - POITypes*
-//      - POI*
-// *(POI = Point of Interest)
-// ============================================================================ //
+// [Models]
 
-// Represents User of the administrative website
 model User {
-    uid      Int @id @default(autoincrement())
-    username String @unique
-    password String
-}
-
-// Represents Tracker in the Database (Currently: only Oyster3)
-model Tracker {
-    uid         String @id @unique //ID (EUI) of the tracker
-    vehicle     Vehicle? @relation(fields: [vehicleId], references: [uid])
-    vehicleId   Int? // Relation Field (n:1)
-    logs        Log[] //Relation Field (n:1)
-    data        Json? // Additional Data
-}
-
-// Represents Data from a tracker at a specific time
-model Log {
-    timestamp   DateTime
-    tracker     Tracker @relation(fields: [trackerId], references: [uid])
-    trackerId   String //Relation Field (1:n)
-    position    Json
-    heading     Float
-    speed       Float
-    battery     Float
-    data        Json? // Additional Data
-
-    @@id([timestamp, trackerId])
+    // this model is ONLY used for the administrative website!
+    username    String @id @unique
+    password    String
```

Comment:
This is probably the hash that is contained in the database, correct? The naming should reflect that (passwordHash or similar).